docs/sandbox-permissions.rst

```diff
@@ -1,40 +1,94 @@
-
-===================================================  ===========================================
-``--filesystem=host``                                Access all files
-``--filesystem=home``                                Access the home directory
-``--filesystem=home:ro``                             Access the home directory, read-only
-``--filesystem=/some/dir --filesystem=~/other/dir``  Access paths
-``--filesystem=xdg-download``                        Access the XDG download directory
-``--nofilesystem=...``                               Undo some of the above
-``--socket=x11 --share=ipc``                         Show windows using X11 [#f1]_
-``--device=dri``                                     OpenGL rendering
-``--socket=wayland``                                 Show windows using Wayland
-``--socket=pulseaudio``                              Play sounds using PulseAudio
-``--share=network``                                  Access the network [#f2]_
-``--talk-name=org.freedesktop.secrets``              Talk to a named service on the session bus
-``--system-talk-name=org.freedesktop.GeoClue2``      Talk to a named service on the system bus
-``--socket=system-bus``                              Unlimited access to all of D-Bus
-===================================================  ===========================================
-
-.. note::
-  Until a sandbox-compatible backend is available, access to dconf needs to be enabled using the following options::
-
-    --filesystem=xdg-run/dconf
-    --filesystem=~/.config/dconf:ro
-    --talk-name=ca.desrt.dconf
-    --env=DCONF_USER_CONFIG_DIR=.config/dconf
-
-.. rubric:: Footnotes
-
-.. [#f1] ``--share=ipc`` means that the sandbox shares IPC namespace with the host. This is not necessarily required, but without it the X shared memory extension will not work, which is very bad for X performance.
-.. [#f2] Giving network access also grants access to all host services listening on abstract Unix sockets (due to how network namespaces work), and these have no permission checks. This unfortunately affects e.g. the X server and the session bus which listens to abstract sockets by default. A secure distribution should disable these and just use regular sockets.
+One of Flatpak's main goals is to increase the security of desktop systems by isolating applications from one another. This is achieved using sandboxing and means that, by default, Flatpak applications have extremely limited access to the host environment. This includes:
 
+- No access to any host files except the runtime, the app and ``~/.var/app/$APPID``. Only the last of these is writable.
+- No access to the network.
+- No access to any device nodes (apart from ``/dev/null``, etc).
+- No access to processes outside the sandbox.
+- Limited syscalls.  For instance, apps can't use nonstandard network socket types or ptrace other processes.
+- Limited access to the session D-Bus instance - an app can only own its own name on the bus.
```

Comment:
and can only talk to the portal.

Reply:
I don't understand this comment. The "portal" as in "portals", or something else?

Reply:
Yes those portals, all apps can talk to them.

Reply:
And are they relevant to D-Bus specifically? It feels a bit out of place to mention them here.